### PR TITLE
[artifactory] Added tomcat acceptCount and maxThreads

### PR DESCRIPTION
**Pull request by DNewt**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Adding Tomcat maxThreads and acceptCounts so that we can tune it for heavy loads.
https://jfrog.com/knowledge-base/how-do-i-tune-artifactory-for-heavy-loads/

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
https://github.com/jfrog/charts/issues/898

**Special notes for your reviewer**:
Sorry I don't know how to add the extraConfig part properly to the values.yaml - please help :(

### CODE REVIEW

File: stable/artifactory-ha/values.yaml [hunk 1]
```diff
@@ -487,9 +490,17 @@ artifactory:
     {{- end }}
       database:
         maxOpenConnections: {{ .Values.artifactory.database.maxOpenConnections }}
+      tomcat:
+        connector:
+          maxThreads: 200
+          extraConfig: "acceptCount=\"100\""
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml:496]: Extract the `extraConfig` also to be a configuration value under `artifactory.tomcat.connector`.

File: stable/artifactory-ha/values.yaml [hunk 2]
```diff
@@ -487,9 +490,17 @@ artifactory:
     {{- end }}
       database:
         maxOpenConnections: {{ .Values.artifactory.database.maxOpenConnections }}
+      tomcat:
+        connector:
+          maxThreads: 200
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml:495]: Should be `maxThreads: {{ .Values.artifactory.tomcat.connector.maxThreads }}`

File: stable/artifactory-ha/values.yaml [hunk 3]
```diff
@@ -1134,6 +1145,9 @@ artifactory:
 access:
   database:
     maxOpenConnections: 80
+  tomcat:
+    connector:
+      maxThreads: 200
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml:1150]: Access default is `50`. Stick to the default.

File: stable/artifactory-ha/values.yaml [hunk 4]
```diff
@@ -487,9 +490,17 @@ artifactory:
...
+        connector:
+          maxThreads: 200
+          extraConfig: "acceptCount=\"100\""
     access:
       database:
         maxOpenConnections: {{ .Values.access.database.maxOpenConnections }}
+      tomcat:
+        connector:
+          maxThreads: 200
+          extraConfig: "acceptCount=\"100\""
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml:503]: Add to `access.tomcat.connector`

File: stable/artifactory-ha/values.yaml [hunk 5]
```diff
@@ -487,9 +490,17 @@ artifactory:
...
     access:
       database:
         maxOpenConnections: {{ .Values.access.database.maxOpenConnections }}
+      tomcat:
+        connector:
+          maxThreads: 200
```
Comment (eldada) [inline stable/artifactory-ha/values.yaml:502]: Should be maxThreads: {{ .Values.access.tomcat.connector.maxThreads }}

File: stable/artifactory/README.md [hunk 6]
```diff
@@ -996,8 +996,10 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.ssh.internalPort`            | Artifactory SSH internal port | `1339`                                      |
 | `artifactory.ssh.externalPort`            | Artifactory SSH external port | `1339`                                     |
 | `artifactory.terminationGracePeriodSeconds`   | Termination grace period (seconds)  | `30s`                                |
+| `artifactory.tomcat.connector.maxThreads`         | Maximum number of threads allowed to run at any given time   | `200` |
```
Comment (eldada) [inline stable/artifactory/README.md:999]: It's max number of connections to Artifactory connector. Not number of threads.

File: stable/artifactory/README.md [hunk 7]
```diff
@@ -996,8 +996,10 @@ The following table lists the configurable parameters of the artifactory chart a
...
 | `artifactory.systemYaml`                      | Artifactory system configuration (`system.yaml`) as described here - https://www.jfrog.com/confluence/display/JFROG/Artifactory+System+YAML  | `see values.yaml`    |
 | `access.database.maxOpenConnections`                      | Maximum amount of open connections from Access to the DB  | `80`    |
+| `access.tomcat.connector.maxThreads`         | Maximum number of threads allowed to run at any given time   | `200` |
```
Comment (eldada) [inline stable/artifactory/README.md:1002]: It's max number of connections to Access connector. Not number of threads.

File: stable/artifactory-ha/values.yaml (hunk 1 above)
Comment (DNewt) [reply stable/artifactory-ha/values.yaml:496]: Hey eldada, thank you for helping me out. 
I am a bit confused about this one as here (I may be misunderstanding what you mean): https://www.jfrog.com/confluence/display/JFROG/Artifactory+System+YAML
the connector does not have acceptCount as an option but maintenance Connector does.

Also I was told to use extra config here: https://github.com/jfrog/charts/issues/898

Thanks again for your help

Comment (eldada) [reply stable/artifactory-ha/values.yaml:496]: Yes. This is correct. I might have confused you with my comment. 
You still need the `extraConfig` here. Just its value should come from the values.yaml's `artifactory.tomcat.connector`. 
So here, you'll have 
```
extraConfig: {{ .Values.artifactory.tomcat.connector.extraConfig }}
```

and higher in the values.yaml, you'll have

```
artifactory:
  tomcat:
    connector:
      maxThreads: 200
      extraConfig: `acceptCount="100"`
```
Need to test it... the formatting might need a fixup.

Comment (DNewt) [reply stable/artifactory-ha/values.yaml]: This should be resolved now thank you. 
I am a little unsure about my changes to the ReadMe (is my wording correct for acceptCount?)

**Pull request merged**